README.md

```diff
@@ -190,6 +192,20 @@ If you have installed Jina, the above flowchart can be generated via:
 jina export flowchart flow.yml flow.svg
 ```
 
+
+### Stable Diffusion weights
+
+If you want to use Stable Diffusion, you will first need to register an account on the website [Huggingface](https://huggingface.co/) and agree to the terms and conditions for the model. You can find the version of the model required by going here:
```

Comment:
```suggestion
If you want to use Stable Diffusion, you will first need to register an account on the website [Huggingface](https://huggingface.co/), go to the [repo](https://huggingface.co/CompVis/stable-diffusion-v-1-4-original), and agree to the terms and conditions for the model. You can find the version of the model required by going here:
```

If I go to directly download page without accepting terms, it show `403` page.

Reply:
I will update the README with clearer instructions.